src/XMakeBuildEngine/Logging/BaseConsoleLogger.cs

```diff
@@ -370,7 +370,7 @@ internal bool IsVerbosityAtLeast(LoggerVerbosity checkVerbosity)
         /// <summary>
         /// Sets foreground color to color specified
         /// </summary>
-        internal static void SetColor(ConsoleColor c)
+        public static void SetColor(ConsoleColor c)
```

Comment:
See: Logging\ConsoleLogger.cs line 84 and 85. Make it a protected method doesn't work either.